testing/test_hellsemble/hellsemble.py

```diff
@@ -351,12 +364,89 @@ def evaluate_hellsemble(
         }
 
         if callable(self.metric):
-            y_pred_proba = self.predict_proba(X)[:, 1]
-            return self.metric(y, y_pred_proba)
+            y_pred = self.predict(X)
+            return self.metric(y, y_pred)
 
         if self.metric == "roc_auc":
             y_pred_proba = self.predict_proba(X)[:, 1]
             return metrics_map["roc_auc"](y, y_pred_proba)
 
         y_pred = self.predict(X)
         return metrics_map[self.metric](y, y_pred)
+
+    def get_progressive_scores(
```

Comment:
exception during this function will break (I guess) the whole instance (estimators will not be saved). It would be better to do it without modifing original attributes as it is supposed to be read-only function. If it's hard to do it, just please add `try-except` logic to return to the original structure of estimators

Reply:
hard to do without modifying original list, thats why I save it to new variable, not in self, I'll add try except logic